relation_engine_server/test/test_json_validation.py

```diff
@@ -350,7 +350,7 @@ def test_date_format_validation(self, schema_arg=None, schema_file_arg=None):
 
         # pyyaml-specific issue: dates get automatically parsed into datetime objects (doh!)
         file_path = os_path.join(json_validation_dir, "unquoted_date.yaml")
-        err_str = "datetime.date\(2020, 6, 6\) is not of type 'string'"
+        err_str = r"datetime.date\(2020, 6, 6\) is not of type 'string'"
```

Comment:
you could also use `re.escape` which is probably more robust than escaping each individual test string